### DISCUSSION TITLE
Microlags and Warnings in MP

### DISCUSSION THREAD

**Discussion by altbier1** [Mogli12/AIVehicleExtension#274] [Previous version]
During  playing MP we have microlags and lua callstacks from the fieldjobmanager if we use the AIVehicleExtension (AI-VE). After turning off the AI-VE we were able to drive as normal, if we use the AI-VE we had something like stop and go. We have also warnings from the Particle system:
2020-05-05 16:17 Warning: Particle system emitter mesh 'particlesBackShape' must be marked as a CPU Mesh or Emitter geometry name must contain 'emit' if it is set from script
After turning off the AI-VE the warning appears not...

**Comment by Mogli12** [comment]
Please check version 0.0.7.1
